.github/workflows/publish.yml

```diff
@@ -19,6 +19,8 @@ jobs:
   publish-gpr:
     needs: build
     runs-on: ubuntu-latest
+    permissions:
+      packages: write
```

Comment:
https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token